spec/controllers/api/v0/business_days_controller_spec.rb

```diff
@@ -9,6 +9,7 @@
     end
 
     it 'should handle holidays correctly' do
+      BusinessTime::Config.holidays << Date.parse('2016-09-05')
```

Comment:
can we put this in a separate PR? seems unrelated and I'd like to have it as its own thing